<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_key">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>384</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_input_key">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>key：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Password generation</string>
    </property>
    <widget class="QPushButton" name="pushButton_enc_psw">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>GetPassword</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mac">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MAC：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mac">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_sn">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>56</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SN：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_info">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>details：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enc_file">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clipboard">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>copy psw</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Decrypt file</string>
    </property>
    <widget class="QPushButton" name="pushButton_open_file">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open file</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_file_path">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>25</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dec_file">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_file_string">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>24</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>File：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_open_dir">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Directory</string>
     </property>
    </widget>
   </widget>
   <zorder>lineEdit_key</zorder>
   <zorder>label_input_key</zorder>
   <zorder>groupBox</zorder>
   <zorder>plainTextEdit_info</zorder>
   <zorder>label_show_info</zorder>
   <zorder>pushButton_enc_file</zorder>
   <zorder>pushButton_clipboard</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>pushButton_open_dir</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
